lib/src/footer/types.ts

```diff
@@ -0,0 +1,71 @@
+type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type Size = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type Padding = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
```

Comment:
Maybe is a good idea to unify in a unique type.

Reply:
Done